glean_parser/swift.py

```diff
@@ -107,6 +107,18 @@ def variable_name(var: str) -> str:
         return var
 
 
+class Metric:
+    """
+    Data struct holding information about a metric to be used in the template.
+    """
+
+    category_name = None  # type: str
+    objs = None  # type: Dict[str, Union[metrics.Metric, pings.Ping]]
+    is_ping_type = False  # type: bool
+
+    pass
```

Comment:
`pass` isn't necessary, since there are some members defined.

Reply:
Ah, before I didn't have members so I needed it.